.env

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_PROJECT_ID=38b236298d3253f98d42e08991ad74d3
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)